README.md

```diff
@@ -76,6 +76,36 @@ Most of the bindings ([`src/auto`](src/auto)) are generated by [gir](https://git
 
 When opening a PR please put the changes to the `src/auto` directory in a separate commit.
 
+## Contributors
+
+### Code Contributors
```

Comment:
This one is wrong considering that a lot of gtk-rs contributors never contributed to this repository. It'd put aside a lot of them, which seems unfair. I think it'd be better to just remove this section.